impress-org/givewp
# Feature/resolve paypal commerce quick issues

Resolves
- #4985
- #4986
- Set default font to `Arial`. [Slack discussion](https://givewp.slack.com/archives/C015XD31J5S/p1596228118005400) 
- Moved assets from PayPalCommerce directory to assets/src/


## Description
In this PR, I resolved minor issues related to the PayPal Commerce feature.
`Arial` is default web font and assign by user agent stylesheet if not any font using to DOM element. To assign `Arial` font to hosted card fields, I did not assign `font-family` these fields, so default stylesheet assigns `Arial` font.

## Affects
This Pr does not affect existing functionality. A noticeable change is I moved PayPal Commerce assets to `assets/src` directory.

## Visuals
![image](https://user-images.githubusercontent.com/1784821/89293425-88a41e80-d67b-11ea-9b72-6d2c6926e908.png)
![image](https://user-images.githubusercontent.com/1784821/89293450-93f74a00-d67b-11ea-8729-813bddf95f7d.png)
![image](https://user-images.githubusercontent.com/1784821/89293541-b7ba9000-d67b-11ea-8c32-63ee936e929f.png)
![image](https://user-images.githubusercontent.com/1784821/89293561-c012cb00-d67b-11ea-90f9-7e9975c82cfe.png)


## Pre-review Checklist

## Testing Instructions
- To check PayPal Commerce create order and capture payment log, you have to request PayPal to return an error and you can do it by adding support error constant in the header. Please instruction give in this pr description: https://github.com/impress-org/givewp/pull/4977
- To check the transaction payment link, open any successful donation detail page and view `Transaction Id` in the right sidebar.
